### ISSUE TITLE
[Gallery][iOS] Commandbar- Header 'Material Page 1 and 2' is missing.

### ISSUE THREAD

**Issue by sakshi173** [unoplatform/uno#9385]
### Current behavior

![commandbar issue- Photos](https://user-images.githubusercontent.com/74563602/181508367-f4838cfd-4b8c-469b-b499-3fa2926f6bb8.png)

### Expected behavior

![commandbar correct‎- Photos](https://user-images.githubusercontent.com/74563602/181508384-3a199be4-dcdd-4fc1-af26-4e4d85e6fc1d.png)

### How to reproduce it (as minimally and precisely as possible)

1. Open Uno Gallery APP
2. Tap Burger Menu on the left top right.
3. Tap "CommandBar" from the "Component" list.
4. Tap on the 'Show Sample' button.
5. Observe that the header 'Material Page 1 is missing.
6. Click on 'Navigater to next page' option.
7. Observe that the header 'Material Page 2 is missing.

### Affected platforms

iOS

**Comment by sakshi173** [comment]
@jeromelaban, @kazo0 , Still this issue is present, please look into it.

**Comment by kazo0** [comment]
@sakshi173 is it failing for the Canary? This should be fixed with the latest packages

**Comment by sakshi173** [comment]
@kazo0 , Yes it failing for the canary build. Below is the screenshot from the latest canary build for iOS.
![Commandbar issue](https://github.com/unoplatform/uno/assets/74563602/b4ebf4ab-027e-470f-a246-105f1f0ca80a)

**Comment by kazo0** [comment]
Hmm that's not good, will take a look today

**Comment by agneszitte** [comment]
@sakshi173 can you please re-test this issue to see if it is still the case, please

**Comment by sakshi173** [comment]
@agneszitte and @kazo0 still this issue is present.

**Comment by MartinZikmund** [comment]
Only relevant to native
